gui-libs/wlroots/wlroots-9999.ebuild

```diff
@@ -60,20 +65,20 @@ BDEPEND="
 "
 
 src_configure() {
+	local backends="$(usex drm 'drm,' '')"
+	backends+="$(usex libinput 'libinput,' '')"
+	backends+="$(usex x11-backend 'x11,' '')"
+	# Get rid of a trailing comma
+	backends="${backends%,}"
 	# xcb-util-errors is not on Gentoo Repository (and upstream seems inactive?)
 	local emesonargs=(
-		"-Dxcb-errors=disabled"
 		-Dxcb-errors=$(usex xcb-errors enabled disabled)
 		$(meson_use tinywl examples)
 		-Drenderers=$(usex vulkan 'gles2,vulkan' gles2)
 		-Dxwayland=$(usex X enabled disabled)
-		-Dbackends=drm,libinput$(usex x11-backend ',x11' '')
+		-Dbackends="${backends}"
+		-Dsession=$(usex session enabled disabled)
```

Comment:
You should be able to use meson_feature or meson_use here instead (for all of these args).

Reply:
Is meson already inherited? Oh, it is. Well, will do then

Reply:
Done in the latest commit (did not touch wlroots-0.16.2.ebuild, though)

Reply:
Thanks, this looks really good. I hope to see more contributions from you in future!